clang/docs/ReleaseNotes.rst

```diff
@@ -647,6 +647,8 @@ Improvements to Clang's diagnostics
 
 - Clang now shows implicit deduction guides when diagnosing overload resolution failure. #GH92393.
 
+- Clang now emits ``-Wnewline-eof`` when the last newline is deleted by a preceding backslash. #GH41571.
```

Comment:
```suggestion
- Clang now emits ``-Wnewline-eof`` when the last newline follows a splicing backslash . #GH41571.
```

Reply:
Is the space before the full stop intentional?